### PR TITLE
Rename trigger to _ver_avoid_drop_of_versioned_table

### PR DESCRIPTION
**Pull request by strk**
Note I've added a DROP TRIGGER IF EXISTS for the old
name, otherwise continuously upgraded databases may
end up with a trigger which is NEVER removed on upgrade

### CODE REVIEW

File: sql/01-enable_versioning.sql [hunk 1]
```diff
@@ -228,6 +228,11 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
-DROP EVENT TRIGGER IF EXISTS _ver_abort_on_drop_of_versioned_table;
-CREATE EVENT TRIGGER _ver_abort_on_drop_of_versioned_table
+-- This is only needed on databases which ever installed a
```
Comment (imincik) [inline sql/01-enable_versioning.sql]: Hi @strk , I there was no deployment done with dev version. Please drop this command

File: sql/01-enable_versioning.sql [hunk 2]
```diff
@@ -228,6 +228,11 @@ BEGIN
...
+-- development version of table_version, which at one point
+-- created a trigger named "_ver_abort_drop_of_versioned_table"
+DROP EVENT TRIGGER IF EXISTS _ver_abort_drop_of_versioned_table;
+
+DROP EVENT TRIGGER IF EXISTS _ver_avoid_drop_of_versioned_table;
```
Comment (imincik) [inline sql/01-enable_versioning.sql]: I like more `_ver_abort_drop_of_versioned_table` than `_ver_avoid_drop_of_versioned_table`

Comment (imincik) [reply sql/01-enable_versioning.sql]: I think we don't need to worry about backwards compatibility of changes in this patch. As I wrote above, there is no prod or any other significant deployment done from master branch.

**Comment by strk** [comment]
Ok @imincik I've called both function and event with `abort`, and rebased against master.
No effort is taken to fix database which ever installed in-dev version of table_version from current master

**Pull request merged**